Backend/Controllers/AuthenticationController.cs

```diff
@@ -79,4 +80,35 @@ public IQueryable<Session> GetSessions([FromAuthentication] User user)
     [HttpGet]
     public User GetAuthUser([FromAuthentication] User user) => user;
 
+    [HttpPost("reset/start")]
+    public async Task InitPasswordReset(string email) {
+        email = email.ToLowerInvariant();
+        var user = await _context.Users.FirstOrDefaultAsync(x => x.Email == email);
+        if (user == null) { return; }
```

Comment:
Just to make sure: Confirmation message is going to be supplied by FE and not BE? I'm not sure what's best practise here since it's going to be a "email has been sent" in any case. Probably doesn't matter all to much. I guess by depending on a message from BE you make sure that the request has been handled in any way.

Reply:
Generally it's better to avoid differentiating between failure and success in a method like this to prevent a possible attacker from gathering information about which emails are signed up on the platform. (This is in contradiction to the ability to search by email, but here I believe it to be appropriate to not respond with a 404.)

Reply:
I know, thats not what I was talking about. Just the question where the message is coming from, not differentiating between failure and success.

Reply:
What message?

Reply:
"email has been sent"
but as i said, doesnt really matter